filter_plugins/with_isolated_workers.py

```diff
@@ -0,0 +1,56 @@
+"""
+Copyright (c) 2018 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+from copy import deepcopy
+import re
+
+
+# Negative regex used to exclude characters that are not allowed
+# in naming a kubernetes resource
+INVALID_KUBERNETES_NAME_CHARS = re.compile(r'[^a-z0-9\.-]+')
+
+
+class FilterModule(object):
+    def filters(self):
+        return {'with_isolated_workers': do_with_isolated_workers}
+
+
+def do_with_isolated_workers(reactor_configs):
+    """Generate reactor configs for each worker cluster
+
+    :param reactor_configs: list<dict>, each dict should contain a name and
+    a data key. The value of name key is used to name the config map object
+    and the value of data key is a reactor config
+
+    :return: a new list of reactor configs that contains new reactor configs
+    for each worker cluster in addition to the original reactor configs
+    """
+    all_configs = list(reactor_configs)
+
+    for config in reactor_configs:
+        clusters = config.get('data', {}).get('clusters', {})
+        for arch, workers_info in clusters.items():
+            for worker_info in workers_info:
+                worker_info = deepcopy(worker_info)
+                worker_info['enabled'] = True
```

Comment:
why do we overwrite 'enabled' value?

Reply:
This is the worker specific reactor config map. It will only contain a single worker cluster.
In some cases, you may want to disable some worker clusters in the main reactor config.
This gives the ability to test the disabled worker cluster if needed.

If you're using a reactor config for a specific worker cluster, that cluster should always be enabled in that particular reactor config.

Reply:
ahh ok got it